### ISSUE TITLE
Code Panel: weird syntax highlighting

### ISSUE THREAD

**Issue by jamessspanggg** [reposense/RepoSense#1116]
[Link to reproduce](https://deploy-preview-1106--reposense.netlify.com/#search=&sort=groupTitle&sortWithin=title&since=2017-10-09&timeframe=commit&mergegroup=false&groupSelect=groupByRepos&breakdown=false&tabOpen=true&tabType=authorship&tabAuthor=ongspxm&tabRepo=reposense%2FRepoSense%5Bmaster%5D)

<img width="464" alt="Screen Shot 2020-03-04 at 11 43 55 PM" src="https://user-images.githubusercontent.com/32864116/75896438-103df500-5e72-11ea-81e2-242d9043cd24.png">

**Comment by Tejas2805** [comment]
Based on my investigation its looks like, `hljs` is picking it up as regex and that's why the wrong highlighting.

**Comment by jamessspanggg** [comment]
@Tejas2805 I've looked into it and I think the issue is because -

``${(pos - offset) * 100 / base}%`; is an ES6 way for string interpolation, while the code highlighted is based on javascript instead of ES6:
<img width="899" alt="Screen Shot 2020-03-05 at 8 50 29 PM" src="https://user-images.githubusercontent.com/32864116/75983188-00cab480-5f23-11ea-9413-40fa83d5a061.png">

Have tried switching to interpret it as ES6 and it highlights correctly:
<img width="1213" alt="Screen Shot 2020-03-05 at 8 51 35 PM" src="https://user-images.githubusercontent.com/32864116/75983249-2061dd00-5f23-11ea-94a3-42c7ae8869c6.png">

**Comment by gerhean** [comment]
> UPDATE: This is a bug in highlight.js and happens because it's picking up the code as regex. I have tried the latest released version (v9.18.1), but it's still not fixed. They plan to release it in v10.0

The latest Highlight.js is at v10.7.1, so can be fixed easily. However the given code does not exist anymore so can't really test.

**Comment by dcshzj** [comment]
The bug still seem to exist even in v10.7.1. You can reproduce this by specifying the date range to generate the report:

```
gradlew run -Dargs="--since d1 --until 16/03/2020"
gradlew run -Dargs="--view ./reposense-report"
```

Click on the Code Panel for Metta (ongspxm) to see the weird syntax highlighting.

We should prepare a custom report and submit an upstream task, so that they are able to fix this.

**Comment by gerhean** [comment]
![image](https://user-images.githubusercontent.com/16630400/113386146-2f33f100-93bc-11eb-8cc1-f597a10eaf0f.png)

Fixed by v10.7.1

Will be doing PR soon.
